cpu/native/can/candev_linux.c

```diff
@@ -195,7 +196,14 @@ static int _init(candev_t *candev)
 
     real_bind(dev->sock, (struct sockaddr *)&addr, sizeof(addr));
 
-    _set_bittiming(dev, &candev->bittiming);
+    /* Only set bitrate on real can interfaces.
+     * Not supported on virtual can interfaces ("vcanX") */
+    if (strncmp(dev->conf->interface_name, "can", strlen("can"))) {
+        LOG_INFO("not setting bitrate on virtual can interface %s\n", dev->conf->interface_name);
```

Comment:
I would not mix different types of debug output, i.e. please use `DEBUG` as throughout this file. Also could use same message as below

Reply:
Aha, had already thought about it, but I assumed that LOG_INFO did something extra over DEBUG, hence the request to use that type. 
I'll fix it straight away!

Reply:
its okay to use `LOG_INFO` and similar in general, but as said I would rather not mix them with `DEBUG`.